README.rst

```diff
@@ -60,6 +77,9 @@ Testing is done via tox to test all supported versions:
 
     $ tox
 
+(Docker users should run tox inside a bash shell in the running container.
+Start the shell using "docker container exec -it xblock-sdk_xblock_sdk_1 bash")
+
```

Comment:
I think it would be better if instead of this, at line 71 we added "If using docker, all these commands need to be run inside the xblock-sdk container."?

Reply:
Great suggestion. Done.